### PR TITLE
Initial working version of the CaVEMan wrapper container

### PR DESCRIPTION
**Pull request by drjsanger**

### CODE REVIEW

**Comment by keiranmraine** [comment]
@rulixxx this looks very good.  Have you attempted to use the container with a small data set (I'm guessing you don't have the information to)?

**Comment by rulixxx** [comment]
@keiranmraine  not yet.  I guess that was going to be the next step after getting the image to work in Singularity (its the first time I use Singularity).

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+FROM ubuntu:16.04
+
+LABEL maintainer="cgphelp@sanger.ac.uk" \
+      uk.ac.sanger.cgp="Cancer, Ageing and Somatic Mutation, Wellcome Trust Sanger Institute" \
+      version="v1.0.0" \
+      description="cgpCaVEManWrapper docker"
+
+RUN apt-get -yq update
+RUN apt-get install -yq --no-install-recommends \
```
Comment (byb121) [inline Dockerfile]: Does this also install python? I saw python is installed in the build stage, but not here. Is python only required for some tool to build but not after installation?

Comment (keiranmraine) [reply Dockerfile]: Good spot, I don't think python is needed in this stack...

**Comment by drjsanger** [comment]
Aside from confirming whether python is required or not this looks good.

**Pull request merged**